app/views/tag/_tagging.html.erb

```diff
@@ -37,39 +77,17 @@ $(".label").on("click", function(e){
       }
     }
   });
-</script>
-
-<% parent ||= nil %>
-
-<% if current_user && (parent != :profile || (current_user.id == @user.id || current_user.role == "admin")) %>
-<% url = url || "/tag/create/" + @node.id.to_s %>
-
-<form id="tagform" class="form" data-remote="true" action="<%= url %>">
-  <div class="control-group">
-    <input name="remote" type="hidden" value="true" />
-    <div class="input-group">
-      <span class="input-group-addon"><i class="fa fa-tags"></i></span>
-      <input autocomplete="off" class="tag-input form-control" name="name" type="text" placeholder="<%= t('tag._tagging.enter_tags') %>" data-provide="typeahead" />
-      <div class="input-group-btn">
-        <a class="btn btn-default blurred-location-input" rel="tooltip" title="Link with a location">
-          <i class="fa fa-map-marker"></i>
-        </a>
-        <%= render partial: 'tag/advanced_tagging' %>
-      </div>
-    </div>
-    <script>
-      jQuery(document).ready(function() {
-        <% if @node %>
-          // node tags:
-          initTagForm("/tag/delete/<%= @node.id %>", '#tagform');
-        <% else %>
-          // user tags:
-          initTagForm("/profile/tags/delete/<%= @user.id %>", '#tagform');
-        <% end %>
-      });
-    </script>
-  </div>
-</form>
-<% end %>
+var arr = new Array();
```

Comment:
Here, i'm cautious. This code is used in multiple places, including for user tags on the profile. I wonder if we need to isolate this further from the rest of the code here to make sure it works. Let me expand on this in a new comment though.